tests/sentry/ownership/test_grammar.py

```diff
@@ -190,14 +192,14 @@ def test_matcher_test_stacktrace() -> None:
             "frames": [{"filename": "foo/file.py"}, {"abs_path": "/usr/local/src/other/app.py"}]
         }
     }
+    munged_data = Matcher.munge_if_needed(data)
 
-    assert Matcher("path", "*.py").test(data)
-    assert Matcher("path", "foo/*.py").test(data)
-    assert Matcher("path", "/usr/local/src/*/app.py").test(data)
-    assert not Matcher("path", "*.js").test(data)
-    assert not Matcher("path", "*.jsx").test(data)
-    assert not Matcher("url", "*.py").test(data)
-    assert not Matcher("path", "*.py").test({})
```

Comment:
This assertion got copy/pasta'd in a handful of tests, it's still present on line 166.